docs/guide/0-compose.md

```diff
@@ -108,3 +108,13 @@ You can check the following informations below:
 
 - [ulcl-example branch](https://github.com/free5gc/free5gc-compose/tree/ulcl-example), or
 - [patch file](https://github.com/ianchen0119/free5gc-compose-ulcl)
+
+## Run N3IWUE 
+You can access the N3IWUE bash and run it:
```

Comment:
## [Optional] Run N3IWUE
If you want to use N3IWUE, you can access the N3IWUE container and run N3IWUE using the following commands.